documentation/docs/11-ssr-and-javascript.md

```diff
@@ -64,12 +64,16 @@ In many cases, you'll only want to prerender specific pages in your app. You'll
 
 The prerenderer will start at the root of your app and generate HTML for any prerenderable pages it finds. Each page is scanned for `<a>` elements that point to other pages that are candidates for prerendering — because of this, you generally don't need to specify which pages should be accessed. If you _do_ need to specify which pages should be accessed by the prerenderer, you can do so with the `pages` option in the [prerender configuration](#configuration-prerender).
 
+One benefit of fully prerendering a site is that you do not need to maintain / pay for NodeJS servers to perform SSR to serve personalized pages. SSR happens only at deploy time, some people call this Server Side Generatin (SSG), to differentiate from per user SSR. Once generated, these sites are typically served from CDNs, leading to great "Time to first byte" performance. Search engines can crawl the prerendered static pages on CDNs, which can lead to better rankings. Make sure to not prerender pages you do not want search engines to see.
+
+This delivery model has a nickname: JAMStack. While static sites are clear candidates for this model, prerendered sites are not limited to static content. You can build authenticated, dynamic, personalized sites in this model, just make sure all user specific data is fetched and rendered client-side. Pages can be prerendered, with blanks or placeholders, to be replaced on the client with personalized data fetched in onMount() from backend API endpoints, either your own, or provided by 3rd parties. This also works well for PWAs, prerendered pages can be cached on user's device, and as user navigates from page to page, the app can fetch() data needed to personalize the page.
```

Comment:
I still feel that this and the next paragraph are not matching the flow of the rest of the docs. It's a short dive into what JAMstack is, why it's good, what's possible etc. I don't think this belongs in here.